terracumber_config/tf_files/Uyuni-Master-NUE.tf

```diff
@@ -128,21 +128,23 @@ module "cucumber_testsuite" {
       }
     }
     server = {
+      image = "sles15sp4o"
```

Comment:
Be caution, Uyuni use opensuse153o for server and proxy.
It did not move yet to opensuse154o

Reply:
Fixed.

Reply:
I am wondering here though if we then should update the minions to openSUSE Leap 15.3 at all if we do not plan to use 15.4, yet. If we test the update from 15.2 to 15.3 would't it make sense to migrate this to 15.3 -> 15.4?

Reply:
To be honest, I'm not sure what we do for Uyuni, we will need to review the code and see if we make any differentiation between Uyuni and SUMA on this feature.